index.html

```diff
@@ -105,6 +106,46 @@ <h2 class="header-3 color-n800 mb-12">Tonic</h2>
         </ul>
         <div class="action">
           <button id="myBtn" type="button" class="button cardButton myBtn1">See Project</button>
+
+          <!-- modal -->
+          <div id="myModal" class="modal">
```

Comment:
- [ ] **Please make sure that the Modal popup data is populated with the data from the projects array.**

Please use the projects array mentioned in the previous comment to populate data in your modals.

Reply:
Dear @smcommits,

I have fixed it. Please, check.

MR

Elmar